### PR TITLE
don't review this i'm just hacking on reviews stuff

### PR DESCRIPTION
**Pull request by jdhollen**
**Related issues**: N/A

### CODE REVIEW

File: server/remote_cache/byte_stream_client/BUILD [hunk 1]
```diff
@@ -20,6 +21,7 @@ go_library(
         "//server/util/ziputil",
         "@org_golang_google_genproto_googleapis_bytestream//:bytestream",
         "@org_golang_google_grpc//:go_default_library",
+        "@org_golang_google_grpc//metadata",
         "@org_golang_google_protobuf//proto",
```
Comment (jdhollen) [inline server/remote_cache/byte_stream_client/BUILD:25]: cc

File: server/remote_cache/byte_stream_client/BUILD [hunk 2]
```diff
@@ -11,6 +11,7 @@ go_library(
         "//server/endpoint_urls/cache_api_url",
         "//server/environment",
         "//server/remote_cache/digest",
+        "//server/util/authutil",
```
[the remark this answers is not in the document]
Comment (jdhollen) [reply server/remote_cache/byte_stream_client/BUILD:14]: abcd

File: server/remote_cache/byte_stream_client/BUILD [hunk 3]
```diff
@@ -11,6 +11,7 @@ go_library(
...
         "//server/util/flag",
```
Comment (jdhollen) [inline server/remote_cache/byte_stream_client/BUILD:15]: qq

File: server/remote_cache/byte_stream_client/BUILD [hunk 4]
```diff
@@ -20,6 +21,7 @@ go_library(
         "//server/util/ziputil",
```
Comment (jdhollen) [inline server/remote_cache/byte_stream_client/BUILD:21]: i don't like ziputil

File: server/remote_cache/byte_stream_client/BUILD [hunk 5]
```diff
@@ -20,6 +21,7 @@ go_library(
...
         "@org_golang_google_genproto_googleapis_bytestream//:bytestream",
         "@org_golang_google_grpc//:go_default_library",
```
[the remark this answers is not in the document]
Comment (jdhollen) [reply server/remote_cache/byte_stream_client/BUILD:23]: c

File: server/remote_cache/byte_stream_client/BUILD (hunk 2 above)
[the remark this answers is not in the document]
Comment (iain-macdonald) [reply server/remote_cache/byte_stream_client/BUILD:14]: ship it

File: server/remote_cache/byte_stream_client/BUILD [hunk 6]
```diff
@@ -11,6 +11,7 @@ go_library(
         "//server/endpoint_urls/cache_api_url",
         "//server/environment",
```
Comment (jdhollen) [inline server/remote_cache/byte_stream_client/BUILD:12]: testing, again.

**Review by jdhollen — COMMENTED** [review]
I am testing, again.

**Pull request closed without merging**